docs/index.html

```diff
@@ -0,0 +1,341 @@
+<!DOCTYPE html>
+<!-- 
+  All Rights Reserved
+  This software is property information of AKUREY S.A.
+  Contact us at contact@akurey.com
+  @summary short description for the file
+-->
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto Slab"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Task 03 - One Piece Website</title>
+
+    <script>
+      function toggleCollapse(elementId) {
+        const elem = document.getElementById(elementId);
+        if (elem.classList.contains("show")) {
+          elem.classList.remove("show");
+        } else {
+          elem.classList.add("show");
+        }
+      }
+
+      function toggleSipnosis(elementId) {
+        toggleCollapse(elementId);
+
+        if (!document.getElementById(elementId).classList.contains("show")) {
```

Comment:
No, always the sense makes sense, said that, find a good/bad solution as a workaround and communicate what's going on and how you solved it but the time. 👍🏼